packages/editor/src/components/provider/index.js

```diff
@@ -30,6 +32,19 @@ class EditorProvider extends Component {
 		}
 	}
 
+	componentDidMount() {
+		if ( ! this.props.settings.styles ) {
+			return;
+		}
+		postcss( [ wrap( { selector: '.editor-block-list__block' } ) ] )
```

Comment:
Would it make sense for us to have a new class/set of classes we use for this purpose? It might make it easy to disable theme styles or pinpoint what is styling the editor.

I don't know if that'll ruin our specificity; I figured if the class was on the same `div` as `.editor-block-list__block` it'd be fine, but I forget if that's the case. 🤷‍♂️

Reply:
Yes, maybe a new class makes sense. Something clear like `.editor-styles-wrapper` or something like that.